src/bentoml/_internal/cloud/deployment.py

```diff
@@ -296,7 +299,13 @@ def get_bento_info(
     if project_path:
         from bentoml.bentos import build_bentofile
 
-        bento_obj = build_bentofile(build_ctx=project_path, _bento_store=_bento_store)
+        console = Console()
+        with console.status(f"🍱 Building bento from project: {project_path}"):
```

Comment:
Can we reuse the `self.spinner` for this status?(`rich.status` is just a wrapper of Live(Spinner))

Reply:
‌‌I feel that the Spinner class we encapsulated is not very user-friendly. This part will become quite ugly.

![image](https://github.com/bentoml/BentoML/assets/1206493/c24e5ec7-1bc9-4a35-a5d5-5ce0db4f94f8)

Reply:
<img width="1340" alt="image" src="https://github.com/bentoml/BentoML/assets/1206493/99837783-53ee-4110-94fe-b9c94ed695b0">

This way it can be expressed the same as console.status 🥶